assets/src/components/StyledTextField.js

```diff
@@ -10,7 +10,6 @@ const StyledTextField = withStyles({
       borderBottomColor: '#ffae42'
     },
     '& .MuiInputBase-input': {
-      marginTop: '10px',
       color: 'green'
```

Comment:
This is how it impacts the change
![Screen Shot 2020-01-31 at 10 57 33 AM](https://user-images.githubusercontent.com/8579775/73553853-a36db000-4418-11ea-946f-dcdb5fc0eb79.png)
![Screen Shot 2020-01-31 at 10 57 40 AM](https://user-images.githubusercontent.com/8579775/73553861-a5d00a00-4418-11ea-9248-9cc7a5bf0cea.png)

Reply:
Thanks!